### ISSUE TITLE
Crash today, very quickly after opening. It crashed so quickly that I was barely able to move the camera.

### ISSUE THREAD

**Issue by NativeJim** [Infixo/CS2-InfoLoom#19]
```
[object Object]
NullReferenceException: Object reference not set to an instance of an object
  at Unity.Entities.ComponentLookup`1[T].get_Item (Unity.Entities.Entity entity) [0x00032] in <42dd0aeaaef34ed8acb4b4fe5f093234>:0 
  at InfoLoom.Systems.WorkplacesInfoLoomUISystem+CalculateWorkplaceDataJob.Execute (Unity.Entities.ArchetypeChunk& chunk, System.Int32 unfilteredChunkIndex, System.Boolean useEnabledMask, Unity.Burst.Intrinsics.v128& chunkEnabledMask) [0x000f8] in C:\Repos\CS2-Mods\CS2-InfoLoom\Systems\WorkplacesInfoLoomUISystem.cs:164 
  at InfoLoom.Systems.WorkplacesInfoLoomUISystem+CalculateWorkplaceDataJob.Unity.Entities.IJobChunk.Execute (Unity.Entities.ArchetypeChunk& chunk, System.Int32 unfilteredChunkIndex, System.Boolean useEnabledMask, Unity.Burst.Intrinsics.v128& chunkEnabledMask) [0x00000] in C:\Repos\CS2-Mods\CS2-InfoLoom\Systems\WorkplacesInfoLoomUISystem.cs:269 
  at Unity.Entities.JobChunkExtensions+JobChunkProducer`1[T].ExecuteInternal (Unity.Entities.JobChunkExtensions+JobChunkWrapper`1[T]& jobWrapper, System.IntPtr bufferRangePatchData, Unity.Jobs.LowLevel.Unsafe.JobRanges& ranges, System.Int32 jobIndex) [0x000b5] in <42dd0aeaaef34ed8acb4b4fe5f093234>:0 
  at Unity.Entities.JobChunkExtensions+JobChunkProducer`1[T].Execute (Unity.Entities.JobChunkExtensions+JobChunkWrapper`1[T]& jobWrapper, System.IntPtr additionalPtr, System.IntPtr bufferRangePatchData, Unity.Jobs.LowLevel.Unsafe.JobRanges& ranges, System.Int32 jobIndex) [0x00000] in <42dd0aeaaef34ed8acb4b4fe5f093234>:0 
  at (wrapper delegate-invoke) Unity.Entities.JobChunkExtensions+JobChunkProducer`1+ExecuteJobFunction[InfoLoom.Systems.WorkplacesInfoLoomUISystem+CalculateWorkplaceDataJob].invoke_void_JobChunkExtensions/JobChunkWrapper`1<T>&_intptr_intptr_JobRanges&_int(Unity.Entities.JobChunkExtensions/JobChunkWrapper`1<InfoLoom.Systems.WorkplacesInfoLoomUISystem/CalculateWorkplaceDataJob>&,intptr,intptr,Unity.Jobs.LowLevel.Unsafe.JobRanges&,int)
Colossal.Logging.CustomLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```

**Comment by Infixo** [comment]
See #16 and #17 
It is a rare bug that happens probably after bulldozing. Also, one person was using ZoneToolkit and it probably (?) was somehow related. The problem is that this code is from CO (I refactored one of their systems) and it **should** work unless there is some internal data corruption.

**Comment by khallmark** [comment]
@Infixo should be able to close this :-D

**Comment by Infixo** [comment]
Fixed in #22
